rubix-core/pom.xml

```diff
@@ -15,19 +15,15 @@
 
     <properties>
         <main.basedir>${project.parent.basedir}</main.basedir>
+        <dep.hadoop2.version>2.6.0</dep.hadoop2.version>
```

Comment:
this is not required, it is provided by root pom

Reply:
Done